presto-docs/src/main/sphinx/language/reserved.rst

```diff
@@ -78,4 +78,5 @@ Keyword                        SQL:2016      SQL-92
 ``WHEN``                       reserved      reserved
 ``WHERE``                      reserved      reserved
 ``WITH``                       reserved      reserved
+``PERSISTENT``
```

Comment:
I'd vote for no new syntax (but if the syntax stays, this list is alphabetically ordered)